docs/source/installation.rst

```diff
@@ -32,7 +32,7 @@ Install
 
 1. Install the package. ::
 
-    $ pip install git+https://github.com/unt-libraries/django-premis-event-service@1.2.5
```

Comment:
Maybe we should remove the version so that we don't have to update this every time the version gets bumped.